### PR TITLE
Modified eqemu_server.pl to pull latest PEQ

### PR DESCRIPTION
**Pull request by alimalkhalifa**
* Also added an escape to skip pulling Maps (tighter feedback loop if
developing)

### CODE REVIEW

File: utils/scripts/eqemu_server.pl [hunk 1]
```diff
@@ -1705,26 +1709,22 @@ sub fetch_server_dlls {
...
     find(
         sub { push @files, $File::Find::name unless -d; },
         $start_dir
     );
     for my $file (@files) {
         $destination_file = $file;
-        $destination_file =~ s/updates_staged\/peq_db\///g;
-        if ($file =~ /peqbeta|player_tables/i) {
+        $destination_file =~ s/updates_staged\/peq_db\/peq-dump\///g;
+        if ($file =~ /create_tables_content|create_tables_login|create_tables_player|create_tables_queryserv|create_tables_state|create_tables_system/i) {
```
Comment (Akkadius) [inline utils/scripts/eqemu_server.pl]: Why not simply run the `create_tables_all` here ?

File: utils/scripts/eqemu_server.pl [hunk 2]
```diff
@@ -1705,26 +1709,22 @@ sub fetch_server_dlls {
...
             print "[Install] DB :: Installing :: " . $destination_file . "\n";
             get_mysql_result_from_file($file);
         }
     }
-
-    #::: PEQ DB baseline version
-    print get_mysql_result("DELETE FROM db_version");
-    print get_mysql_result("INSERT INTO `db_version` (`version`) VALUES (9130);");
```
Comment (Akkadius) [inline utils/scripts/eqemu_server.pl]: I'm glad you caught this

File: utils/scripts/eqemu_server.pl (hunk 1 above)
Comment (alimalkhalifa) [reply utils/scripts/eqemu_server.pl]: I had it doing all but ran into pwd issues. Am I missing something? 😑

Comment (Akkadius) [reply utils/scripts/eqemu_server.pl]: You need to be in the context of the directory to source the `all` file as it is referring to files in the same directory context

This is fine

Did you test a full server install with this and did a cursory functionality test?

Comment (alimalkhalifa) [reply utils/scripts/eqemu_server.pl]: Yup, my dev box is running this install and all works fine so far!

Comment (alimalkhalifa) [reply utils/scripts/eqemu_server.pl]: 1. Tested windows only though

**Pull request merged**